docs/user-guide.md

```diff
@@ -102,7 +102,7 @@ See: [`Ethereum Whitepaper`](https://github.com/ethereum/wiki/wiki/White-Paper#m
 
 Ethereum contracts or `smart-contracts` are computational code stored on the Ethereum blockchain. Contracts can be written in a higher level language like [Solidity](https://solidity.readthedocs.io/en/develop/) which then compiles down into EVM (Ethereum Virtual Machine) bytecode that can be stored on the chain. To use or deploy these contracts with `ethjs` you need the **ABI** and (if your deploying) the **bytecode** or (if your just using it) the **address**. Contracts can be designed to send and receive, process and store data and `ether` from other `accounts` or `contracts`. `ethjs` provides a `eth.contract` object to help you interact with and deploy Ethereum contracts (its design is very similar to its `web3.js` counterpart).
 
-See: [`Browser-Solidity`](https://ethereum.github.io/browser-solidity/) an in browser Solidity IDE for building contracts, [`Solidity Read The Docs`](https://solidity.readthedocs.io/en/develop/), [`ethjs-contract`](https://github.com/ethjs/ethjs-contract), [`Ethereum Whitepaper`](https://github.com/ethereum/wiki/wiki/White-Paper#applications). Note, this will actually provide you with the necessary contract `bytecode` and `ABI` required to deploy and use the contracts.
+See: [`Browser-Solidity`](https://ethereum.github.io/browser-solidity/) an in browser Solidity IDE for building contracts, [`Solidity Read The Docs`](https://solidity.readthedocs.io/en/develop/), [`ethjs-contract`](https://github.com/MetaMask/ethjs-contract), [`Ethereum Whitepaper`](https://github.com/ethereum/wiki/wiki/White-Paper#applications). Note, this will actually provide you with the necessary contract `bytecode` and `ABI` required to deploy and use the contracts.
```

Comment:
```suggestion
See: [`Remix IDE`](http://remix.ethereum.org) an in browser Solidity IDE for building contracts, [`Solidity Read The Docs`](https://solidity.readthedocs.io/en/develop/), [`ethjs-contract`](https://github.com/MetaMask/ethjs-contract), [`Ethereum Whitepaper`](https://github.com/ethereum/wiki/wiki/White-Paper#applications). Note, this will actually provide you with the necessary contract `bytecode` and `ABI` required to deploy and use the contracts.
```